sdk/digitaltwins/Azure.ResourceManager.DigitalTwins/src/Customized/DigitalTwinsDescriptionResource.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Azure.Core;
+using System.Threading.Tasks;
+using System.Threading;
+
+namespace Azure.ResourceManager.DigitalTwins
+{
+    public partial class DigitalTwinsDescriptionResource : ArmResource
+    {
+        /// <summary>
+        /// Delete a DigitalTwinsInstance.
+        /// <list type="bullet">
+        /// <item>
+        /// <term>Request Path</term>
+        /// <description>/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DigitalTwins/digitalTwinsInstances/{resourceName}</description>
+        /// </item>
+        /// <item>
+        /// <term>Operation Id</term>
+        /// <description>DigitalTwins_Delete</description>
+        /// </item>
+        /// </list>
+        /// </summary>
+        /// <param name="waitUntil"> <see cref="WaitUntil.Completed"/> if the method should wait to return until the long-running operation has completed on the service; <see cref="WaitUntil.Started"/> if it should return after starting the operation. For more information on long-running operations, please see <see href="https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/samples/LongRunningOperations.md"> Azure.Core Long-Running Operation samples</see>. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        public virtual async Task<ArmOperation<DigitalTwinsDescriptionResource>> DeleteAsync(WaitUntil waitUntil, CancellationToken cancellationToken = default)
+        {
+            using var scope = _digitalTwinsDescriptionDigitalTwinsClientDiagnostics.CreateScope("DigitalTwinsDescriptionResource.Delete");
+            scope.Start();
+            try
+            {
+                var response = await _digitalTwinsDescriptionDigitalTwinsRestClient.DeleteAsync(Id.SubscriptionId, Id.ResourceGroupName, Id.Name, cancellationToken).ConfigureAwait(false);
+                var operation = new DigitalTwinsArmOperation<DigitalTwinsDescriptionResource>(Response.FromValue(new DigitalTwinsDescriptionResource(Client, response), response.GetRawResponse()));
```

Comment:
We should not add a Get request in delete request.
Instead we could change like this:
```suggestion
                var operation = new DigitalTwinsArmOperation<DigitalTwinsDescriptionResource>(Response.FromValue(this, response.GetRawResponse()));
```